heudiconv/info.py

```diff
@@ -11,7 +11,7 @@
 REQUIRES = [
     'nibabel',
     'pydicom',
-    'nipype>=1.0.0',
+    'nipype @ git+https://github.com/effigies/nipype.git@0196beac33d0c8c992bcc07b83c0b368810be02d',
```

Comment:
I think we'll want to leave nipype > 1.0.0 but add the pin for 2.7 only
```suggestion
    'nipype @ git+https://github.com/effigies/nipype.git@0196beac33d0c8c992bcc07b83c0b368810be02d; python_version == "2.7"',
```

Reply:
I think the correct way would be:

```suggestion
    'nipype >=1.0.0; python_version > "3.0"',
    'nipype >=1.0.0,!=1.2.1,!=1.2.2; python_version == "2.7"',
```

Reply:
yep, I like that better and it won't require another PR 😄